common/test/acceptance/pages/lms/course_page.py

```diff
@@ -38,3 +39,14 @@ def has_tab(self, tab_name):
         """
         tab_nav = TabNavPage(self.browser)
         return tab_name in tab_nav.tab_names
+
+    def skip_to_main_content(self):
+        """
+        Ensures skip links function and focus on the intended href
+        """
+
+        skip_to = self.q(css=".nav-skip").attrs('href')[0]
+        self.q(css=".nav-skip").first().click()
+        return EmptyPromise(
+            lambda: self.q(skip_to).present, "Main content area received focus"
```

Comment:
You may also want to use [self.wait_for](http://bok-choy.readthedocs.org/en/latest/api_reference.html#bok_choy.page_object.PageObject.wait_for) instead of `EmptyPromise`.  They both have the same result, but `wait_for` is a bit more clear.